server/db/models.js

```diff
@@ -0,0 +1,39 @@
+const Sequelize = require('sequelize');
+const sequelize = new Sequelize('tbd', 'root', '12345');
+
+const users = sequelize.define('user', {
+  userName: {
+    type: Sequelize.STRING
+  },
+  password: {
+    type: Sequelize.STRING
+  },
+  facebookId: {
+    type: Sequelize.STRING
+  },
+  token: {
+    type: Sequelize.STRING
+  }
+}, {
+  tableName: 'users',
+  timestamps: false,
+});
+
+sequelize
+  .authenticate()
+  .then(err => {
+    console.log('sqlz Connection has been established successfully.');
+  })
+  .catch(err => {
+    console.log('sqlz Unable to connect to the database:', err);
+  });
+
+sequelize
+  .sync({ force: false })
```

Comment:
We don't want to run `DROP TABLE IF EXISTS` every time. I think we need an init & reset script instead.